src/deepsparse/transformers/utils/token_generator.py

```diff
@@ -168,5 +168,5 @@ def _update_frequencies(self, token: numpy.ndarray):
 
     def _initialize_token_frequencies(self):
         unique_tokens, frequencies = numpy.unique(self.tokens, return_counts=True)
-        for token, frequnecies in zip(unique_tokens, frequencies):
-            self.token_frequencies[token] += frequnecies
+        for token, freq in zip(unique_tokens, frequencies):
+            self.token_frequencies[token] += freq
```

Comment:
nit: always prefer to write out variable names unless it forces bad line breaks

Reply:
this was just a quick intervention to 1) fix the existing typo 2) avoid reusing the existing `frequencies` variable name